src-tauri/tauri.conf.json

```diff
@@ -6,7 +6,7 @@
     "distDir": "../dist"
   },
   "package": {
-    "productName": "Project-Raphael",
+    "productName": "Project Raphael",
```

Comment:
I might be wrong, but product names shouldn't have spaces in them because some programs don't accept spaces in a path name. We can keep "Project Raphael" for now and change it if we find an issue, this is only a comment.